MycroftAI/mycroft-core
# Add msk

## Description
This adds [mycroft-skills-kit](github.com/mycroftai/mycroft-skills-kit) to the requirements so that the executable is available after running `source .venv/bin/activate`.

## How to test
 - Run `./dev_setup.sh`
 - `source .venv/bin/activate`
 - Make sure `msk -h` or `msk create` works
